README.md

```diff
@@ -92,6 +92,15 @@ The `--input` flag can also be a good way to see how different input types would
 conftest parse examples/hcl2/terraform.tf -i hcl2
 ```
 
+#### Multi input type
+
+Conftest supports different types of input file type in a single call.
```

Comment:
Maybe it should be slightly reworded: `` `conftest` supports multiple different input types in a single call:``. The second line is already implied by the `--input` section above.

Reply:
Thanks for your advice.